numpyro/distributions/discrete.py

```diff
@@ -33,8 +34,39 @@ def _entropy(self, n, p):
 
 
 class _bernoulli_gen(jax_discrete, bernoulli_gen):
+    def __new__(cls, *args, **kwargs):
+        return super(_bernoulli_gen, cls).__new__(cls)
+
+    def __init__(self, *args, **kwargs):
+        self.is_logits = kwargs.pop("is_logits", False)
+        super(_bernoulli_gen, self).__init__(*args, **kwargs)
+
+    def freeze(self, *args, **kwargs):
+        self._ctor_param.update(is_logits=kwargs.pop("is_logits", False))
```

Comment:
We update `_ctor_param` so that [frozen distribution](https://github.com/scipy/scipy/blob/master/scipy/stats/_distn_infrastructure.py#L427) makes correct signatures for its self.dist.

Reply:
This is a nice work-around.